ch-06/src/Solution618.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Scanner;
+
+public class Solution618 {
+    public static void main(String[] args) {
+        final Scanner input = new Scanner(System.in);
+        final String password = input.nextLine();
+
+        passwordChecker(password);
+    }
+    public static boolean isPasswordLongEnough(String str) {
+        return str.length() >= 10;

```

Comment:
Although we're not familiar with class fields yet, it might be a good idea to extract the magic number `10` into a constant. Something like

~~~~Java
final int MIN_ACCEPTED_PASSWORD_LENGTH = 10;

return str.length() >= MIN_ACCEPTED_PASSWORD_LENGTH ;
~~~~